WordPress/wporg-main-2022
# Add About landing page

Closes #148

Props @tellyworth for initial content

Note that the design uses Courier Prime for the monospace serif font, but our theme has 'IBM Plex Mono', so that's what I'm using for now. I have [notified design](https://dotorgdesign.wordpress.com/2022/11/04/about-redesign/#comment-1727).

### Screenshots

| Desktop | Tablet | Mobile |
|--------|-------|-------|
| ![localhost_8888_about-2_(Desktop) (1)](https://user-images.githubusercontent.com/1017872/213332309-6be32f13-7581-479a-90b8-d4f7d920e6f2.png) | ![localhost_8888_about-2_(iPad) (1)](https://user-images.githubusercontent.com/1017872/213332325-c6031178-d586-493d-b185-0236d35094d4.png) | ![localhost_8888_about-2_(Samsung Galaxy S20 Ultra) (3)](https://user-images.githubusercontent.com/1017872/213576616-eebf4723-07ec-4c3b-8b54-ee5bd8633b36.png) |


### How to test the changes in this Pull Request:

1. Build the styles
2. Add a page at `/about-2`
3. Open the page, test responsive behaviour, links, cross browser layout. The cover in particular should fluidly scale between 600px and 1320px
